spec/lib/models/locales_updater_configuration_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require_relative '../../../lib/open_project/translations/models/locales_updater'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.